source/components/monthly-log-component.js

```diff
@@ -0,0 +1,167 @@
+            letter-spacing: -0.015em;
+            color: #F0EBCC;
+            background-color: #344FA1;
+            border-radius: 6px;
+            padding-left: 12px;
+            padding-right: 12px;
+            padding-top: 12px;
+            padding-bottom: 12px;
+            margin-left: 25px;
+            margin-right: 150px;
+            margin-bottom: 30px;
+            max-width: 350px;
+          }
+              
+          button, input[type="submit"], input[type="reset"] {
+            /* Remove default styling */
+            background: none;
+            cursor: pointer;
+            border: none;
+            padding: 0;
+            outline: inherit;
+      
+            /* Add custom styling */
+            transition: background-color 0.5s;
+          }
+
+          button:hover {
+            color: #fffbe3;
+            background-color: #5f52d8;
+          }
+
+          button:active {
+            color: #fffbe3;
+            background-color: #5f52d8;
+          }
+
+          @media (min-width: 800px) {
+            .log-form-section {
+              margin-top: 3em;
+              margin-left: 3em;
+            }
+          }
+
+          @media (max-width: 599px) {
+            .log-title-text {
+              margin-left: 1em;
+            }
+          }
+
+          </style>
+          <section class="log">
+            <section class="log-title-section">
+              <h2 class="log-title-text"><h2>
+            </section>
+          <section class="log-main-section">
+            <section class="log-calendar-section">
+              <h2 class="log-calendar-month-text"></h2>
+                <img class="log-calendar-image-placeholder" 
+                  src="./media/calendar-icon.svg"
+                  alt="please enter a valid path">
+            </section>
+            <section class="log-form-section">
+              <ul class="log-bullets">
+                <li>2: laundry</li>
+                <li>14: grocery shopping</li>
+              </ul>
+              <button class="add-bullet-button">+ add</button>
+            </section>
+          </section>
+          `;
+
+        this.attachShadow({ mode: 'open' })
+        this.shadowRoot.appendChild(template.content.cloneNode(true));
+
+        // Get attributes
+        const title = this.getAttribute('title');
+        const month = this.getAttribute('month');
+
+        // Apply css classes based on value of attributes
+        if (title) {
+            this.shadowRoot.querySelector('.log-title-text').innerText = title;
+        } else {
+            this.shadowRoot.querySelector('.log-title-text').innerText = "Viv's January"
+        }
+
+        if (month) {
+            this.shadowRoot.querySelector('.log-calendar-month-text').innerText = month;
+        } else {
+            this.shadowRoot.querySelector('.log-calendar-month-text').innerText = "JAN";
+        }
+    }
+}
+
+customElements.define('log-component', logComponent);
```

Comment:
- Replace `log-component` with `monthly-log-component`
- Replace logComponent with MonthlyLogComponent

Reply:
@rahuldadlani03 Hi! I added the naming edits you mentioned in the comments and also applied them to the CSS classes as well, just appending 'monthly' to each CSS class. My bad, renaming the class itself slipped my mind. I requested review again. Thanks!